### PR TITLE
Generate ECMAScript module for contrib

### PR DESCRIPTION
**Pull request by ylemkimon**

### CODE REVIEW

File: contrib/copy-tex/copy-tex.webpack.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
+ * This is the webpack entry point for KaTeX. As ECMAScript doesn't support
+ * CSS modules natively, a separate entry point is used.
+ */
+import './copy-tex.css';
+import './copy-tex.js';
```
Comment (kevinbarabash) [inline contrib/copy-tex/copy-tex.webpack.js]: This change doesn't seem strictly necessary.  Was thine thinking to make it follow the same pattern as the main katex build?

File: rollup.config.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+    input: entry.replace('.webpack', ''),
     output: {
-        file: 'dist/katex.mjs',
+        file: `dist/${name}.mjs`,
         format: 'es',
     },
     plugins: [
         babel({runtimeHelpers: true}),
+        alias({
+            katex: '../katex.mjs',
```
Comment (kevinbarabash) [inline rollup.config.js]: Does this require `katex` to appear first in `targets`?  Also, where is this used?

File: webpack.common.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ const targets /*: Array<Target> */ = [
     },
     {
         name: 'contrib/copy-tex',
-        entry: './contrib/copy-tex/copy-tex.js',
+        entry: './contrib/copy-tex/copy-tex.webpack.js',
```
Comment (kevinbarabash) [inline webpack.common.js]: If we're using `rollup.config.js` to build all of these targets, why is `copy-tex` the only one that needed to be changed to use `.webpack.js`?

File: rollup.config.js (hunk 2 above)
Comment (ylemkimon) [reply rollup.config.js]: @kevinbarabash It transforms `import katex from ‘katex’` to `import katex from ‘../katex.mjs’` as browsers cannot resolve a module by its name by spec.

File: webpack.common.js (hunk 3 above)
Comment (ylemkimon) [reply webpack.common.js]: @kevinbarabash `copy-tex` imports a CSS module that ESM doesn’t support. So we use separate entry point for webpack.

File: rollup.config.js (hunk 2 above)
Comment (kevinbarabash) [reply rollup.config.js]: After I asked the question I ended up having to so something similar for another project.

**Pull request merged**